tests/scripts/check-files.py

```diff
@@ -113,16 +119,21 @@ class Utf8BomIssueTracker(FileIssueTracker):
 
     heading = "UTF-8 BOM present:"
 
+    files_exemptions = frozenset([".vcxproj", ".sln"])
```

Comment:
Oh, this looks like the removal of the BOM in the previous commit wasn't quite intentional, then.